NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh

```diff
@@ -1,16 +1,8 @@
 #!/bin/bash
 set -x
 basedir=$( cd "$(dirname "$0")" ; pwd -P )
-conda env update -n base --solver=libmamba -f ${basedir}/env_main_ADE.yaml
+# Install dependencies
+conda env update -n r -f ${basedir}/env_main_ADE.yaml
```

Comment:
minor: for consistency with the `conda run` command that follows, consider spelling out `--name`:
```suggestion
conda env update --name r -f ${basedir}/env_main_ADE.yaml
```

Reply:
fixed